src/Directory.Packages.props

```diff
@@ -64,5 +64,7 @@
     <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="6.0.29" />
     <PackageVersion Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
     <PackageVersion Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
+    <!--Once dotnet restore properly resolves that 'Npgsql' 7.0.7 is not vulnerable, set this ref to 7.0.7.-->
```

Comment:
Why do we want to revert back to 7.0.7 after this resolution instead of just keeping it at 8.0.3?

Reply:
this update utilizes JsonSerialization changes from .NET8 and causes SerializeDeserialize tests to fail in .NET6.